pkg/controller/migration/convert/convert_test.go

```diff
@@ -16,10 +19,21 @@ import (
 
 var _ = Describe("Parser", func() {
 	var ctx = context.Background()
+	var fakeCrd crdv1client.CrdV1Interface
+	BeforeEach(func() {
+		fakeCrd = fakecrd.NewSimpleClientset().CrdV1()
```

Comment:
I think you can remove the duplicate call in the beforeeach if you call this during initialization
```suggestion
	var fakeCrd  = fakecrd.NewSimpleClientset().CrdV1()
	BeforeEach(func() {
```

Reply:
Are you suggesting only have 1 fakecrd.NewSimpleClientset() in the BeforeEach (which I agree with) or are you suggesting that I move the call out of BeforeEach all together?

Reply:
Right, just the former. I found it odd having two instances of `fakeCrd = fakecrd.NewSimpleClientset().CrdV1()` inside the beforeeach.